azure/image/ubuntu/azure-ubuntu.pkr.hcl

```diff
@@ -21,6 +21,7 @@ source "azure-arm" "ubuntu" {
     image_name           = "${var.az_gallery_image_name}"
     image_version        = "${var.az_gallery_image_version}"
     storage_account_type = "Standard_LRS"
+    replication_regions  = ["eastus" , "westus"]
```

Comment:
do we need westus for an e2e test?

Reply:
This is specific to packer version i guess, replication region is not required.

Reply:
yeah, I mean we might be able to shorten the build time/czt resource use by not replicating it to westus